src/main.rs

```diff
@@ -1649,6 +1657,76 @@ fn cmd_regenerate_audit_as(
     Ok(())
 }
 
+fn cmd_renew(out: &Arc<dyn Out>, cfg: &Config, sub_args: &RenewArgs) -> Result<(), miette::Report> {
+    trace!("renewing wildcard audits");
+    let mut store = Store::acquire_offline(cfg)?;
+    do_cmd_renew(out, &mut store, sub_args);
+    store.commit()?;
+    Ok(())
+}
+
+fn do_cmd_renew(out: &Arc<dyn Out>, store: &mut Store, sub_args: &RenewArgs) {
+    assert!(sub_args.expiring ^ sub_args.crate_name.is_some());
+
+    let today = <chrono::DateTime<chrono::Utc>>::from(SystemTime::now()).date_naive();
+    let new_end_date = today + chrono::Months::new(12);
+
+    let renew_entry = |entry: &mut WildcardEntry| {
+        entry.end = new_end_date.into();
+    };
```

Comment:
If the entry has `suggest-renewal = false` we should perhaps not renew it? Say there was a previous manager for a crate X, and you had them publishing the crate until they leave the company, then someone new takes over, so you mark the former employee's audit entry as `suggest-renewal = false` so it doesn't get renewed - it's kinda inconvenient that the renewal is happening automatically for both audits in that case.

Perhaps the member should instead be `renew = false` or something like that, and it can prevent the renew command from operating on the entry at all?

Reply:
I thought I had handled this case, and it turns it was partially handled. It would only ignore the `suggest-renewal` in the case where a crate name was supplied. But still, best to put a check in the function.